_info/Role_items

```diff
@@ -0,0 +1,5 @@
+Role_items
+
+There are certain items in the game, which are unobtainable, except by specific roles. That is, items that some roles get due to a certain role power.
+These items are not placed in the player's inventory, and are instead kept track of by the hosts.
```

Comment:
also mention it cant be traded

Reply:
It can't be traded if it's not in the inventory anyway but I'll mention it cause it does seem important